platform_plugin_forum_email_notifier/utils.py

```diff
@@ -32,6 +32,26 @@ def get_subscribers(thread_id):
     return subscribers
 
 
+def get_staff_subscribers(course_id):
+    """Return a list of user ids subscribed to a thread."""
```

Comment:
can we be a bit more specific of what a staff subscriber is here in the docstring?